ui-tests/cypress/integration/content/assets.spec.js

```diff
@@ -122,69 +130,104 @@ describe('Assets', () => {
+              cy.get('@assetToBeDeleted')
+                .then(() => {
+                  page.getContent().getKebabMenu().openDropdown().openEdit();
+                })
+                .then(page => {
+                  page.getContent().getDescriptionInput().clear().type('test');
+                  page.getContent().submit();
+                })
+                .then(page => {
+                  page.getContent().getTableRows().then(rows =>
+                      cy.wrap(rows).children(htmlElements.div).should('contain.text', 'test'));
+                });
+            });
       });
 
       it('Crop image', () => {
-        currentPage = openAssetsPage();
-        cy.wait(2000);
-        cy.get('@assetToBeDeleted')
-          .then(assetToBeDeleted => currentPage.getContent().getKebabMenu(assetToBeDeleted.id).open().openEdit())
-          .then(() => {
-            currentPage.getDialog().getBody().crop(-100, -100);
-            currentPage.getDialog().confirm();
-          });
+        openAssetsPage()
+            .then(page => {
+              cy.get('@assetToBeDeleted')
+                .then(() => {
+                  page.getContent().getKebabMenu().openDropdown().openEdit()
+                      .then(page => {
+                        page.getContent().getImageHeight().invoke('text').then(OriginalImageHeight => {
+                          cy.get('@currentPage')
+                            .then(page => {
+                              page.getContent().openDropDown().openEdit();
+                            })
+                            .then(page => {
+                              page.getAdminDialog().getBody().crop(-50, -50);
+                              page.getAdminDialog().close();
+                              page.getContent().submit();
+                            })
+                            .then(page =>
+                                page.getContent().getKebabMenu().openDropdown().openEdit())
+                            .then(page =>
+                                page.getContent().getImageHeight().invoke('text').then(CroppedImageHeight => {
+                                  expect(CroppedImageHeight).not.equal(OriginalImageHeight);
+                                })
+                            );
+                        });
+                      });
+                });
+            });
 
-        cy.validateToast(currentPage);
       });
 
       it('Rotate image', () => {
-        currentPage = openAssetsPage();
-        cy.wait(2000);
-        cy.get('@assetToBeDeleted')
-          .then(assetToBeDeleted => currentPage.getContent().getKebabMenu(assetToBeDeleted.id).open().openEdit())
-          .then(() => {
-            currentPage.getDialog().getBody().rotate('left');
-            currentPage.getDialog().confirm();
-          });
-
-        cy.validateToast(currentPage);
+        openAssetsPage()
+            .then(page => {
+              cy.get('@assetToBeDeleted')
```

Comment:
It seems that you do not need to do this get...

Reply:
true, i'll remove it